tests/src/common/WskTestHelpers.scala

```diff
@@ -89,10 +75,19 @@ trait WskTestHelpers extends Matchers {
             // delete assets in reverse order so that was created last is deleted first
             val deletedAll = assetsToDeleteAfterTest.reverse map {
                 case ((cli, n, delete)) => n -> Try {
-                    if (delete) {
-                        cli.delete(n)(wskprops)
-                    } else {
-                        cli.sanitize(n)(wskprops)
+                    cli match {
+                        case _: WskPackage =>
+                            val rr = if (delete) cli.delete(n)(wskprops) else cli.sanitize(n)(wskprops)
+                            rr.exitCode match {
+                                case CONFLICT =>
+                                    // retry sanitization on a package since there may be a list (view)
+                                    // operation that requires a retry for eventual consistency
+                                    whisk.utils.retry({
+                                        if (delete) cli.delete(n)(wskprops) else cli.sanitize(n)(wskprops)
```

Comment:
comment says retry sanitize on a package but this will not retry since sanitize does not fail with an exception.
Should the `if (delete)` predicate guard the retry at the outermost level instead?